src/assets/stylesheets/shared/_layout.scss

```diff
@@ -1,371 +0,0 @@
-///////////////////////////
-// Utility styles
-///////////////////////////
-
-@mixin padding-x($count:15) {
-  @for $i from 0 through $count {
-    .padding-#{$i} {
-      padding: #{$i}px !important;
-    }
-    .padding-top-#{$i} {
-      padding-top: #{$i}px !important;
-    }
-    .padding-left-#{$i} {
-      padding-left: #{$i}px !important;
-    }
-    .padding-bottom-#{$i} {
-      padding-bottom: #{$i}px !important;
-    }
-    .padding-right-#{$i} {
-      padding-right: #{$i}px !important;
-    }
-  }
-}
```

Comment:
@dlabrecq this was commented out, but I have added your conversion in its place.